utils/lib/Alert.ts

```diff
@@ -10,6 +10,63 @@ import type {
   QuickstartAlertInput,
 } from '../types/QuickstartMutationVariable';
 import type { QuickstartConfigAlert } from '../types/QuickstartConfig';
+import type { NerdGraphResponseWithLocalErrors } from '../types/nerdgraph';
+
+import {
+  fetchNRGraphqlResults,
+  ErrorOrNerdGraphError,
+} from './nr-graphql-helpers';
+
+import {
+  ALERT_POLICY_REQUIRED_DATA_SOURCES_QUERY,
+  ALERT_POLICY_SET_REQUIRED_DATA_SOURCES_MUTATION,
+} from '../constants';
+
+interface RequiredDataSources {
+  id: string;
+}
+interface AlertPolicy {
+  id: string;
+  metadata: {
```

Comment:
Surprisingly, `metadata` isn't actually required in our GraphQL schema. I think we should assume that it's always going to be present, but I wanted to mention it here.

We should probably update that in our schema at some point.